### PR TITLE
Add test for node to handle duplicate ledger channel

### PR DESCRIPTION
**Pull request by bitwiseguy**
This will allow us to notice if the node engine's code to handle errors without panicking breaks at any point in the future.

### CODE REVIEW

File: node_test/rpc_test.go [hunk 1]
```diff
@@ -143,6 +143,24 @@ func executeNRpcTest(t *testing.T, connectionType transport.TransportType, n int
 	}
 	t.Log("Ledger channels created")
 
+	// try to create duplicate ledger channel to ensure node correctly
+	// handles error without panicking
+	{
+		outcome := simpleOutcome(actors[0].Address(), actors[1].Address(), 100, 100)
+		duplicateLedgerChannel := clients[0].CreateLedgerChannel(actors[1].Address(), 100, outcome)
```
Comment (kerzhner) [inline node_test/rpc_test.go:150]: This name is a bit confusing
```suggestion
		duplicateLedgerChannelObjective := clients[0].CreateLedgerChannel(actors[1].Address(), 100, outcome)
```

**Review by kerzhner — APPROVED** [review]
